packages/marshal/src/rankOrder.js

```diff
@@ -263,6 +263,26 @@ export const makeComparatorKit = (compareRemotables = (_x, _y) => 0) => {
         // If array X is a prefix of array Y, then X has an earlier rank than Y.
         return comparator(left.length, right.length);
       }
+      case 'copyBytes': {
+        const leftArray = new Uint8Array(left.slice());
+        const rightArray = new Uint8Array(right.slice());
+        const byteLen = Math.min(left.byteLength, right.byteLength);
+        for (let i = 0; i < byteLen; i += 1) {
+          const leftByte = leftArray[i];
+          const rightByte = rightArray[i];
+          if (leftByte < rightByte) {
+            return -1;
+          }
+          if (leftByte > rightByte) {
+            return 1;
+          }
+        }
```

Comment:
This really isn't in the stdlib, optimized down to `memcmp`?! (looks... nope...) Wow.

Reply:
Hi @dckc feel free to propose something to tc39. But I'm resolving here ;)